include/effects/strip/misceffects.h

```diff
@@ -260,7 +267,10 @@ class ColorFillEffect : public LEDStripEffect
     {
         if (_everyNth != 1)
           fillSolidOnAllChannels(CRGB::Black);
-        fillSolidOnAllChannels(_color, 0, NUM_LEDS, _everyNth);
+        if (_ignoreGlobalColor == false && g_ptrSystem->DeviceConfig().ApplyGlobalColors() == true)

```

Comment:
I won't hold back approval over this, but I'd consider using the shorter version because the line is already pretty long:

```suggestion
        if (!_ignoreGlobalColor && g_ptrSystem->DeviceConfig().ApplyGlobalColors())
```